src/actions/types.js

```diff
@@ -16,6 +16,7 @@ export const USER_ERROR = "USER_ERROR"
 export const GET_USER = "GET_USER"
 
 // Quiz
+export const SET_COOKIES = "SET_COOKIES"
```

Comment:
The rest of these were previously alphabetized. I have no opinion on whether they should be. (Generally I favor style rules encoded in ESLint.)

Reply:
I just moved it, makes it easy :)